### PR TITLE
fix:[PIE-6468]:execution-history-spec-fix

### PR DESCRIPTION
**Pull request by namanharness**
### Summary

Fixed tests in these files :-
- PipelineTemplate.spec.ts 
- CreatePipelineWithDeploymentTemplate.spec.ts 
- ExecutionHistory.spec.ts

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by chetanvekariya-harness — APPROVED** [review]
Improve PR title. Do not write generic PR titles. Mention about test cases you fixed in short summary. It helps in finding PR, if needed some day.

**Comment by namanharness** [comment]
retrigger titlecheck

**Review by chetanvekariya-harness — APPROVED** [review]

File: cypress/integration/72-templates-library/CreatePipelineWithDeploymentTemplate.spec.ts [hunk 1]
```diff
@@ -113,14 +115,15 @@ describe.skip('Pipeline Template creation and assertion', () => {
     cy.contains('div', 'Pipeline Studio').should('be.visible')
     cy.contains('a', 'Pipeline Studio').should('be.visible')
     cy.contains('p', pipelineMadeFromTemplate).should('be.visible')
-    cy.contains('div', 'Unsaved changes').should('be.visible')
 
-    cy.get('input[name="service"]').click()
+    cy.get('input[name="service"]', { timeout: 10000 }).should('be.visible').click()
     cy.contains('p', 'testService1').should('be.visible').click()
-    cy.wait(1000)
+    cy.contains('p', 'testService1').should('be.visible')
```
Comment (rameshx) [inline cypress/integration/72-templates-library/CreatePipelineWithDeploymentTemplate.spec.ts:121]: `cy.contains('p', 'testService1').should('be.visible')` is also present in line 120

Comment (namanharness) [reply cypress/integration/72-templates-library/CreatePipelineWithDeploymentTemplate.spec.ts:121]: yes, on line 120 we select it from services dropdown.
& on line 121 we assert that the selected service name is visible on service tab

Comment (PratyushGar) [reply cypress/integration/72-templates-library/CreatePipelineWithDeploymentTemplate.spec.ts:121]: You can assert for service selection using `input[value="service1"]`

Comment (namanharness) [reply cypress/integration/72-templates-library/CreatePipelineWithDeploymentTemplate.spec.ts:121]: @PratyushGar that will be asserting service name inside input tag of service,  

I was doing assertion for the 2nd one in image
image

I will add the one you said as well

**Review by rameshx — APPROVED** [review]

**Review by PratyushGar — APPROVED** [review]

**Pull request merged**